lib/server/constants.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-env node */
+
+module.exports.GAME_NOT_FOUND = 100;
```

Comment:
Can be shortened into
```JavaScript
module.exports = {
  GAME_NOT_FOUND: 100,
  ...
};
```
As this file will never export anything but error codes.